tensorflow/contrib/model_pruning/python/layers/core_layers.py

```diff
@@ -72,8 +72,8 @@ class _MaskedConv(base.Layer):
       linear activation.
     use_bias: Boolean, whether the layer uses a bias.
     kernel_initializer: An initializer for the convolution kernel.
-    bias_initializer: An initializer for the bias vector. If None, no bias will
-      be applied.
+    bias_initializer: An initializer for the bias vector. If None, the default
+      initializer will be used.
```

Comment:
I'm not sure this would apply to this file.

Reply:
sorry, let me see again.